_posts/2021-02-23-OpenSSL.markdown

```diff
@@ -0,0 +1,58 @@
+---
+title: Switching back to OpenSSL
+layout: post
+---
+
+The Void Linux team is planning on [performing the switch to
+OpenSSL](https://github.com/void-linux/void-packages/pull/21056) before March
+is over.
+
+For most users, this shouldn't be noticeable, since we will be pushing the same
+packages that already had to be rebuilt twice a year due to LibreSSL ABI
+breaks, but if anything in your setup depends explicitly on LibreSSL (you know
+who you are!), be sure to migrate away from that.
+
+The important parts of the post are done, but you can read on to learn about
```

Comment:
Consider a horizontal rule here, and then a less flippant lead in to the technical discussion.

Reply:
I think dropping this line and replacing with a rule would be enough; the next sentence leads right on to the technical side of the discussion fairly naturally.